hedera-node/src/main/java/com/hedera/services/txns/crypto/CryptoCreateTransitionLogic.java

```diff
@@ -119,6 +134,19 @@ public void doStateTransition() {
 
             txnCtx.setCreated(created);
             txnCtx.setStatus(SUCCESS);
+            if (!op.getAlias().isEmpty()) {
+                aliasManager.link(op.getAlias(), EntityNum.fromAccountId(created));
+            }
+            if (op.getAlias().size() > EVM_ADDRESS_SIZE) {
+                final var key = asPrimitiveKeyUnchecked(op.getAlias());
+                JKey jKey = asFcKeyUnchecked(key);
+                aliasManager.maybeLinkEvmAddress(jKey, EntityNum.fromAccountId(created));
+            }
+            if (op.hasKey() && !op.getKey().getECDSASecp256K1().isEmpty()) {
+                aliasManager.link(
+                        ledger.getAccountsLedger().getImmutableRef(created).getAlias(),
+                        EntityNum.fromAccountId(created));
+            }
         } catch (InsufficientFundsException ife) {
             txnCtx.setStatus(INSUFFICIENT_PAYER_BALANCE);
         } catch (Exception e) {
```

Comment:
I think we'll need to add a catch when we throw `InvalidTransactionException` from an already used alias because currently in these scenarios we come here and `FAIL_INVALID` is set. 

I'm also wondering whether we should check if aliases are in use in the semantic checks 🤔